docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -0,0 +1,266 @@
+   | `--ssh-key` | The path to the SSH key on your local client that matches the SSH key label in your IBM Cloud infrastructure account. | | Required |
+   | `--swarm-name`, `--name` | The name for your swarm and prefix for the names of each node. | d4ic | Required |
+   | `--docker-ee-url` | The Docker EE installation URL associated with your subscription. For beta, you received this in your welcome email. | | Required |
+   | `--manager` | Deploy 1, 3, or 5 manager nodes. | 3 | Optional |
+   | `--workers`, `-w` | Deploy a minimum of 1 and maximum of 10 worker nodes. | 3 | Optional |
+   | `--datacenter`, `-d` | The location (data center) that you deploy the cluster to. Availabe locations are dal12, dal13, fra02, hkg02, lon04, par01, syd01, syd04, tor01, wdc06, wdc07. | wdc07 | Optional |
+   | `--verbose`, `-v` | Enable verbose mode | | Optional |
+   | `--hardware` | If "dedicated" then the nodes are created on hosts with compute instances in the same account. | Shared | Optional |
+   | `--manager-machine-type` | The machine type of the manager nodes: u1c.1x2, u1c.2x4, b1c.4x16, b1c.16x64, b1c.32x128, or b1c.56x242. Higher machine types cost more, but deliver better performance: for example, u1c.2x4 is 2 cores and 4 GB memory, and b1c.56x242 is 56 cores and 242 GB memory. | u1c.2x4 | Optional |
+   | `--worker-machine-type` | The machine type of the worker nodes: u1c.1x2, u1c.2x4, b1c.4x16, b1c.16x64, b1c.32x128, or b1c.56x242. Higher machine types cost more, but deliver better performance: for example, u1c.2x4 is 2 cores and 4 GB memory, and b1c.56x242 is 56 cores and 242 GB memory. | u1c.1x2 | Optional |
+
+4. Create the cluster. Use the `--swarm-name` flag to name your cluster, and fill in the credentials, SSH, and Docker EE installation URL variables with the information that you retrieved before you began.
+    ```bash
+    $ bx d4ic create --swarm-name my_swarm \
+    --sl-user user.name.1234567 \
+    --sl-api-key api-key\
+    --ssh-label my_ssh_label \
+    --ssh-key filepath_to_my_ssh_key \
+    --docker-ee-url my_docker-ee-url
+    ```
+
+   > Tip to set environment variables
+   >
+   > You can set your infrastructure API credentials and Docker EE installation URL as environment variables so that you do not have to include them as options when using `bx d4ic` commands. For example:
+   >
+   > export SOFTLAYER_USERNAME=user.name.1234567
+   > export SOFTLAYER_API_KEY=api-key
+   > export D4IC_DOCKER_EE_URL=my_docker-ee-url
+
+
+5. Note the cluster **Name** and **ID**.
+
+   > Swarm provisioning
+   >
+   > Your cluster is provisioned in two stages, and takes a few minutes to provision. Don't try to modify your cluster just yet!
+   > First, the manager node is deployed. Then, the additional infrastructure resources are deployed, including the worker nodes, DTR nodes, load balancers, subnet, and NFS volume.
+
+   * **Provisioning Stage 1**: Check the status of the manager node:
+   {% raw %}
```

Comment:
This section needs to be indented a tab stop so that it is underneath the 5a sub-bullet.